### ISSUE TITLE
Font weight "heavier" on first page

### ISSUE THREAD

**Issue by tronar** [wkhtmltopdf/wkhtmltopdf#4726]
**wkhtmltopdf version(s) affected**: current

**OS information** both linux and windows behave the same

**Description**
Generating pdf from an html page creates heavier (bolder) text for standard text on page 1.

**How to reproduce**
I can try to create a minimal html, but I'm using it on complex css based pages.

**Expected behavior**
Weight of fonts consistent across pages

**Possible Solution**  
Create a fake first page ?

**Comment by PhilterPaper** [comment]
I don't recall ever hearing about this kind of behavior. I think you're going to have to come up with a minimal test case that shows it, before anyone can try to look. Be sure to check if you have any CSS or JS that _might_ apply to just the first page (including changing fonts between page 1 and 2).

By the way, "current" or "latest" version doesn't mean much. 0.12.5 is probably the most widely used, with 0.12.6 being out there, and 0.13 at least in beta. We also need to know what framework is wrapped around wkHTMLtoPDF (if any), driving it.

**Comment by tronar** [comment]
Ok, sounds fair. Current as in 0.12.5 (with patched qt).
There is nothing forcing the paging in the html, and the problem follows the page break inserted by wkhtmltopdf (e.g. if you change render size from A4 to letter). Quite curious.

**Comment by tronar** [comment]
Well, it happens with a fairly small example. Somehow the presence of an image triggers the effect.
Attached a small .html (as .txt)  that shows the issue. Output of test run and image also included for reference.
[test.txt](https://github.com/wkhtmltopdf/wkhtmltopdf/files/4801727/test.txt)
![arduino](https://user-images.githubusercontent.com/5940494/85082320-ef8d7700-b1a4-11ea-944b-56516388de84.png)
[test.pdf](https://github.com/wkhtmltopdf/wkhtmltopdf/files/4801728/test.pdf)

**Comment by PhilterPaper** [comment]
To be honest, I don't see any real difference in the weight of the text between the two pages. The first page _might_ be a hair darker, although that could be an optical illusion from having a dark image on the page. I tried generating my own PDF from your example, and again the text weights looked the same to me. I even added a line of bold text on each page, and it's clearly heavier than the other line. I don't have TexGyreHeros font, and my default font is TimesNewRoman.

The bottom line is that the first page does not appear to be a full bold version of the second page, although without your font I can't tell for sure. If you hadn't told me to look for darker text, I don't think I would have noticed it. I take it that you've looked at the original HTML in a browser, and there wasn't a bold `<b>` section of text left open?

**Comment by tronar** [comment]
Believe me that with full text on the page, it is very noticeable.
No, it's not bold. It's hevier but not that much.

**Comment by tronar** [comment]
[test.pdf](https://github.com/wkhtmltopdf/wkhtmltopdf/files/4806040/test.pdf)

**Comment by PhilterPaper** [comment]
I just can not see the difference. I even zoomed the scale up to 3200% and measured the width of an "l" on both pages -- the same. They appear to be the same shade of black, and the background appears to be the same. It's not "double-striking" characters (fake bold when the font doesn't include a bold).

I can't go further on this unless other people chime in and say that _they_ are seeing a noticeably darker (bolder) first page. Have you tried looking at the PDF on other readers and on other displays? Maybe there's something odd about your hardware or the reader you're using.

**Comment by tronar** [comment]
Ah, did not occurred to me that the reader could be at fault. I'm using acrobat pro or reader with the same result. I'm attaching an image of the rendering just in case.

![weight](https://user-images.githubusercontent.com/5940494/85180606-38ebce00-b25a-11ea-9de5-79518e3b5ab5.png)

Interesting, acrobat in a tablet does some two stage rendering, and its heavier on first pass and lighter on second, but both pages end up the same.
Chrome on windows also renders them the same. Go figure. But's the only place I've seen this.

**Comment by PhilterPaper** [comment]
I'm using Adobe Acrobat Reader DC on Windows 10 on a laptop, and as I said, I can't really see a difference. I think you're seeing something of different hardware and Reader software interacting in a strange way. Anyway, good luck with this, as I can't reproduce it and so can't help out any further.

**Comment by tronar** [comment]
Indeed, thanks.
